src/main/java/seedu/address/logic/commands/DoneCommand.java

```diff
@@ -60,6 +74,9 @@ public CommandResult execute(Model model) throws CommandException {
         //if task is ONGOING, it must have a driver and eventTime
         freeDriverFromTask(taskToMark.getDriver().get(), taskToMark.getEventTime().get());
 
+        //add rating to driver
+        taskToMark.getDriver().get().addRating(rating);
```

Comment:
The driver is already removed from the task by `freeDriverFromTask`, `.get()` will throw a null pointer exception. Should do this before the freeing.